openslides/mediafiles/static/templates/mediafiles/mediafile-list.html

```diff
@@ -63,9 +63,63 @@ <h1 translate>Files</h1>
               </button>
             </div>
           </nav>
+          <nav ng-show="getType(presentedMediafile) === 'image'" ng-class="getNavStyle(scroll)" class="form-inline">
+            <div class="btn-group">
+              <button class="btn btn-default" ng-click="mediafileToggleFullscreen()" title="{{ 'Toggle fullscreen' | translate }}"
+				ng-class="presentedMediafile.fullscreen ? 'btn-primary' : 'btn-default'">
+                <i class="fa fa-arrows-alt"></i>
+              </button>
+            </div>
+            <div class="btn-group">
+              <button class="btn btn-default" ng-click="mediafileRotate()" title="{{ 'Rotate clockwise' | translate }}">
+                <i class="fa fa-repeat"></i>
+              </button>
+            </div>
+            <div class="btn-group">
```

Comment:
This zooming and scrolling buttons does not work for projected images. For me: you can remove these buttons. Use just the fullscreen and rotate button for images. This is enough.